packages/itwin/tree-widget/src/components/trees/categories-tree/CategoriesTreeButtons.tsx

```diff
@@ -28,11 +28,18 @@ export interface CategoriesTreeHeaderButtonProps extends TreeHeaderButtonProps {
  * Custom hook that creates props required to render `CategoriesTreeComponent` header button.
  * @public
  */
-export function useCategoriesTreeButtonProps({ viewport }: { viewport: Viewport }): Pick<CategoriesTreeHeaderButtonProps, "categories" | "viewport"> {
+export function useCategoriesTreeButtonProps({ viewport }: { viewport: Viewport }): {
```

Comment:
Maybe a small code example could be added to function docs, explaining how the result should be used in general case (e.g. `...buttonProps` should be passed to one of the header buttons and `onCategoriesFiltered` should be passed to `useCategoriesTree`).